app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,26 @@
+<header>
+  <div class="site-frame">
+    <div class="header-top">
+      <div class="header-title">
+        Neue Konzepte der Informatik
+      </div>
+      <div class="header-user">
+        <% if signed_in? %>
+          <%= link_to 'Abmelden', logout_path, method: :delete %>
+        <% else %>
+          <!-- <%= link_to 'Anmelden', login_path %> # -->
+          <!-- <button>Login</button> -->
+          <%= button_to "Login", login_path, class: "header-login-button", method: "get" %>
```

Comment:
We have to investigate why `method: :get` does not work, since it is actually "the way"(TM).